tests/tests.js

```diff
@@ -149,11 +149,27 @@ QUnit.module( "CheckIt Tests", {
 
 beforeEach: function() {
     // Prepare something once for all tests
```

Comment:
Better comment: `// Initialize a fresh `checkit` object and a dummy active calendar for every test`